README.md

```diff
@@ -2,6 +2,8 @@
 
 ![rev](documentation/imgs/rev_logo.png)
 
+## NOTICE: REV HAS MIGRATED THE PRIMARY BRANCH FROM `MASTER` TO `MAIN` ON FEBRUARY 05, 2024
+
```

Comment:
Not sure this should be included in this PR

Reply:
comment removed!